frontend/src/app/recipe/components/detail/recipe-detail.component.html

```diff
@@ -1,35 +1,32 @@
 @if (_recipe$ | async; as recipe) {
-<h1>{{recipe.name}}</h1>
+  <h4>{{ recipe.name }}</h4>
 
-@if (recipe.imageUrl) {
-  <img [src]="recipe.imageUrl">
-} @else {
-  <img src="dummy-image-square-300x300.jpg">
-}
+  @if (recipe.imageUrl) {
+    <img [src]="recipe.imageUrl">
+  } @else {
+    <img src="dummy-image-square-300x300.jpg">
+  }
 
-<div class="row">
-    <div class="col-md-8">
-        <h3>Instructions:</h3>
-        <ul>
-            @for (block of recipe.instructions; track $index) {
-            <p>
-              @for(line of block; track $index){
-                <li>{{ line }}</li>
-              }
-            </p>
-            }
-        </ul>
-    </div>
+  <div class="row">
```

Comment:
Why another row here?

Reply:
It's so on big screens the two list get rendered side by side instead below each other.